pkg/services/audit/task.go

```diff
@@ -23,7 +24,7 @@ type Task struct {
 
 	cnrNodes [][]netmap.NodeInfo
 
-	sgList []oid.ID
+	sgList map[oid.ID]storagegroupSDK.StorageGroup
```

Comment:
Why did you go with a map instead of a slice of compound structures?
I don't see any random access anywhere, so slices are a better option.

Reply:
changed

Reply:
https://github.com/nspcc-dev/neofs-node/pull/1402/commits/3a7adef56ab3238eeb8bbf6079a04ba42c7d492a